pyproject.toml

```diff
@@ -145,7 +145,7 @@ coveralls = "^3.0.1"
 towncrier = "^19.2.0"
 toml = "^0.10.0"
 pep440-version-utils = "^0.3.0"
-pydoc-markdown = "^3.5.0"
+pydoc-markdown = "^3.10.0"
```

Comment:
I've also done [a PR on the base repo](https://github.com/NiklasRosenstein/pydoc-markdown/pull/183) to fix the sorting of the sidebar items, which is currently messed up. Hopefully it gets merged and released in the coming days (cc @tmbo)